File: Examples/Java/SDK/src/main/java/com/aspose/imaging/cloud/examples/manipulate/ConvertImageFormat.java
package com.aspose.imaging.cloud.examples.manipulate;

import com.aspose.imaging.cloud.examples.Common;
import java.io.IOException;
import java.nio.file.Files;
import java.nio.file.Path;
import java.nio.file.StandardCopyOption;

public class ConvertImageFormat {

    public static void main(String... args) throws IOException {
        String input = "Sample1.png";
        String output = "Sample2.jpg";
        Path inputFile = Common.getPath(ConvertImageFormat.class, input);
        Path outputFile = Common.getPath(ConvertImageFormat.class, output);

        String format = "jpg";
        String outPath = "";

        com.aspose.imaging.model.ResponseMessage apiResponse
                = Common.getImagingSdk().PostImageSaveAs(
                        format, 
                        outPath, 
                        inputFile.toFile()
                );
                
        Files.copy(apiResponse.getInputStream(), outputFile, StandardCopyOption.REPLACE_EXISTING);

    }
}

File: Examples/DotNET/SDK/VisualBasic/Manipulate/ConvertImageFormat.vb
'////////////////////////////////////////////////////////////////////////
' Copyright 2001-2015 Aspose Pty Ltd. All Rights Reserved.
'
' This file is part of Aspose.Imaging. The source code in this file
' is only intended as a supplement to the documentation, and is provided
' "as is", without warranty of any kind, either expressed or implied.
'////////////////////////////////////////////////////////////////////////

Imports System
Imports Aspose.Cloud

Namespace Aspose.Imaging.Cloud.Examples.Manipulate
	Friend Class ConvertImageFormat
		Shared Sub Main()
			Dim dataDir As String = Common.GetDataDir(System.Reflection.MethodBase.GetCurrentMethod().DeclaringType)
			Dim input As String = "sample1.png"
			Dim output As String = "output.jpg"

			Common.StorageService.File.UploadFile(dataDir & input, input, storage:= Common.STORAGE)

			Dim outPath As String = "Imaging/" & input
			Common.GetImagingSdk().SaveAs.ExportExistingImageToAnotherFormat(input, ImageFormat.Jpg, outPath, Common.FOLDER, storage:= Common.STORAGE)

			Common.StorageService.File.DownloadFile(outPath, dataDir & output, storage:= Common.STORAGE)
		End Sub
	End Class
End Namespace
